[2024-08-16T14:35:09.644+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-16T14:35:09.661+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: third_dag.third_task manual__2024-08-16T14:35:06.785223+00:00 [queued]>
[2024-08-16T14:35:09.670+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: third_dag.third_task manual__2024-08-16T14:35:06.785223+00:00 [queued]>
[2024-08-16T14:35:09.670+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-16T14:35:09.685+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): third_task> on 2024-08-16 14:35:06.785223+00:00
[2024-08-16T14:35:09.692+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1088) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-16T14:35:09.693+0000] {standard_task_runner.py:72} INFO - Started process 1089 to run task
[2024-08-16T14:35:09.692+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'third_dag', 'third_task', 'manual__2024-08-16T14:35:06.785223+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmprpstcdzk']
[2024-08-16T14:35:09.695+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask third_task
[2024-08-16T14:35:09.749+0000] {task_command.py:467} INFO - Running <TaskInstance: third_dag.third_task manual__2024-08-16T14:35:06.785223+00:00 [running]> on host 93607a8c7584
[2024-08-16T14:35:09.835+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Juancito2085' AIRFLOW_CTX_DAG_ID='third_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T14:35:06.785223+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-16T14:35:06.785223+00:00'
[2024-08-16T14:35:09.836+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-16T14:35:09.855+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-16T14:35:09.856+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo task 3 completed']
[2024-08-16T14:35:09.866+0000] {subprocess.py:86} INFO - Output:
[2024-08-16T14:35:09.867+0000] {subprocess.py:93} INFO - task 3 completed
[2024-08-16T14:35:09.868+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-16T14:35:09.904+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-16T14:35:09.905+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=third_dag, task_id=third_task, run_id=manual__2024-08-16T14:35:06.785223+00:00, execution_date=20240816T143506, start_date=20240816T143509, end_date=20240816T143509
[2024-08-16T14:35:09.949+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-16T14:35:09.971+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-16T14:35:09.979+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
